.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,50 +1,17 @@
-<!--- Use this part of template if you're adding new package. Remove the rest. -->
-<!--- BEGIN -->
+<!--- Please check that your pull request satisfy all requirements -->
```

Comment:
Please don't touch this file, check [this](https://docs.hunter.sh/en/latest/creating-new/update.html) page for update package instructions.

Reply:
Didn't touch PULL_REQUEST_TEMPLATE.md. Don't find the proposed line in the original file.
Did I miss something else?

Reply:
> Didn't touch PULL_REQUEST_TEMPLATE.md

I clearly see the change of `.github/PULL_REQUEST_TEMPLATE.md`, don't you agree? :)

> Did I miss something else?

I guess your work is not based on `ruslo/hunter@master` branch, please check the first step in docs:
```
[hunter]> git checkout master
[hunter]> git checkout -b pr.hunter_box_1
```